trade-finance-manager-ui/server/controllers/case/cancellation/validation/validate-bank-request-date.ts

```diff
@@ -46,7 +46,7 @@ export const validateBankRequestDate = (date: DayMonthYearInput): BankRequestDat
   }
 
   return {
-    bankRequestDate: parsedDate,
```

Comment:
If a user e.g. enters:
Day: 2
Month: 3
Year: 2024

I think it's sensible to store this as the start of the 2/3/2024 in the database, to make sure e.g. if they enter today's date, then the deal cancellation will have taken place immediately on submit. But happy to hear other thoughts here!

Reply:
@BethThomas141 i'm not familiar with what happens with deal cancellation,

Feels strange to change the time of cancellation to be _before_ a user actually cancels the deal.

What happens if we leave the timestamp as is, instead of changing to start of the day?

Reply:
That's a good point, I've changed it back to keep the timestamp to preserve the current time (so removed the `startOfDay` here)